src/js/stores/DialogStore.js

```diff
@@ -0,0 +1,104 @@
+var EventEmitter = require("events").EventEmitter;
+var lazy = require("lazy.js");
+
+var AppDispatcher = require("../AppDispatcher");
+var DialogEvents = require("../events/DialogEvents");
+
+function addDialog(dialogs, message, dialogId) {
+  dialogs.push({id: dialogId, message: message});
+}
+
+function removeDialog(dialogs, dialogId) {
+  dialogs = dialogs.filter(function (dialog) {
+    return dialog.id !== dialogId;
+  });
+}
+
+var DialogStore = lazy(EventEmitter.prototype).extend({
+  dialogs: [],
+  handleUserResponse: function (dialogId, acceptCallback, dismissCallback) {
+    dismissCallback = dismissCallback || function () {};
```

Comment:
I have created a PR to use default parameters https://github.com/mesosphere/marathon-ui/pull/173